core/rint/test/TTabComTests.cxx

```diff
@@ -60,6 +60,15 @@ TEST(TTabComTests, Completion)
    // FIXME: The first call is unsuccessful due to a bug in the TTabCom::Hook
    // on some systems.
    GetCompletions("TH1");
+   std::string expected_th1 = "TH1 TH1C TH1D"
+#ifdef R__HAS_DATAFRAME
+      " TH1DModel"
+#endif
+      " TH1Editor TH1F TH1I TH1K"
+#ifndef R__USE_CXXMODULES
+      " TH1Merger"
```

Comment:
Why is this unavailable for modules?

Reply:
Sorry, it's *available* for modules! :+1: for modules! :-)

Reply:
This is a private class : https://github.com/root-project/root/search?q=TH1Merger&unscoped_q=TH1Merger

The PCH identifier leaks probably through the friend class declaration.

Reply:
> Sorry, it's _available_ for modules! 👍 for modules! :-)

It is available for the PCH -- modules behavior is correct.